pipeline/frontend/metadata/environment.go

```diff
@@ -119,7 +119,7 @@ func (m *Metadata) Environ() map[string]string {
 		// TODO Deprecated, remove in 3.x
 		"CI_COMMIT_URL": m.Curr.ForgeURL,
 	}
-	if m.Curr.Event == EventTag {
+	if strings.HasPrefix(m.Curr.Commit.Ref, "refs/tags/") {
```

Comment:
It would be better to keep both:
```suggestion
	if m.Curr.Event == EventTag || strings.HasPrefix(m.Curr.Commit.Ref, "refs/tags/") {
```

Reply:
Thanks for the review. Did that at first, then tried to keep it simple, maybe a bit too much ;-)

Reply:
technically I don't think there could be a case when tag event would not have ref/tags/ prefix but better is to keep both just in case